### PR TITLE
Fix bug with expired token error response

### PR DESCRIPTION
**Pull request by servingUpAces**

### CODE REVIEW

File: tests/app/authentication/test_authentication.py [hunk 1]
```diff
@@ -224,17 +256,3 @@ def __create_post_token(service_id, request_body):
         secret=get_unsigned_secrets(service_id)[0],
         client_id=str(service_id)
     )
-
-
-def test_should_attach_the_current_api_key_to_current_app(notify_api, sample_service, sample_api_key):
```
Comment (leohemsted) [inline tests/app/authentication/test_authentication.py]: why did you remove this test?

Comment (ashimali) [inline tests/app/authentication/test_authentication.py]: Moved not removed?

**Comment by minglis** [comment]
👍
